File: CS/dxDialog1/ViewModels/DialogViewModel.cs
using DevExpress.Mvvm;

namespace dxDialog1.ViewModels {
    public class DialogViewModel : ViewModelBase {

        public DialogViewModel() { }
        public DialogViewModel(DialogViewModel viewModel) {
            Option1 = viewModel.Option1;
            Option2 = viewModel.Option2;
            Option3 = viewModel.Option3;
        }

        protected bool _Option1;
        public bool Option1
        {
            get { return this._Option1; }
            set { this.SetProperty(ref this._Option1, value, "Option1"); }
        }


        protected bool _Option2;
        public bool Option2
        {
            get { return this._Option2; }
            set { this.SetProperty(ref this._Option2, value, "Option2"); }
        }


        protected bool _Option3;
        public bool Option3
        {
            get { return this._Option3; }
            set { this.SetProperty(ref this._Option3, value, "Option3"); }
        }
        public void RestoreDefaults() {
            this.Option1 = true;
            this.Option2 = false;
            this.Option3 = true;

        }
    }
}

File: VB/dxDialog1/ViewModels/DialogViewModel.vb
Imports DevExpress.Mvvm

Namespace dxDialog1.ViewModels
    Public Class DialogViewModel
        Inherits ViewModelBase

        Public Sub New()
        End Sub
        Public Sub New(ByVal viewModel As DialogViewModel)
            Option1 = viewModel.Option1
            Option2 = viewModel.Option2
            Option3 = viewModel.Option3
        End Sub

        Protected _Option1 As Boolean
        Public Property Option1() As Boolean
            Get
                Return Me._Option1
            End Get
            Set(ByVal value As Boolean)
                Me.SetProperty(Me._Option1, value, "Option1")
            End Set
        End Property


        Protected _Option2 As Boolean
        Public Property Option2() As Boolean
            Get
                Return Me._Option2
            End Get
            Set(ByVal value As Boolean)
                Me.SetProperty(Me._Option2, value, "Option2")
            End Set
        End Property


        Protected _Option3 As Boolean
        Public Property Option3() As Boolean
            Get
                Return Me._Option3
            End Get
            Set(ByVal value As Boolean)
                Me.SetProperty(Me._Option3, value, "Option3")
            End Set
        End Property
        Public Sub RestoreDefaults()
            Me.Option1 = True
            Me.Option2 = False
            Me.Option3 = True

        End Sub
    End Class
End Namespace
